src/Template/Bake/Template/view.ctp

```diff
@@ -73,95 +73,91 @@ $pk = "\$$singularVar->{$primaryKey[0]}";
     }
 %>
     </ul>
-</div>
-<div class="<%= $pluralVar %> view large-10 medium-9 columns">
-    <h2><?= h($<%= $singularVar %>-><%= $displayField %>) ?></h2>
-    <div class="row">
+</nav>
+<div class="<%= $pluralVar %> view large-9 medium-8 columns" id="content">
```

Comment:
Personally, I am not a fan of too many non-js-functional IDs in the HTML code layout. Especially such generic ones, as they need to be unique across the HTML body and there could easily be another content injected somewhere. Is there a reason those IDs have been added? Or could those just be a class instead?